api.raml

```diff
@@ -23,33 +22,61 @@ securitySchemes:
           type: string
 
 
-#
-# Example Endpoints
-#
+/sequenceForGff3/{sequenceType}:
+  displayName: Retrieve sequence for GFF3
+  post:
+    securedBy: header-auth
+    queryParameters:
+      deflineFormat: lib.DeflineFormat
```

Comment:
this bunch of query parameters is strictly about output format.  we could bundle them into an `outputConfig` or similar object